pyamplipi/amplipi.py

```diff
@@ -187,6 +187,34 @@ async def load_preset(self, preset_id: int) -> Status:
     async def announce(self, announcement: Announcement) -> Status:
         response = await self._client.post('announce', announcement.json(**json_ser_kwargs))
         return Status.parse_obj(response)
+    
+    def get_version(self, version_raw: str):
+        version_nums = []
+        tmp = 0
+        for ch in version_raw:
+            if ch.isdigit():
+                tmp = (tmp*10) + int(ch)
+            else:
+                version_nums.append(tmp)
+                tmp = 0
+        version_nums.append(tmp)
+
+        return tuple(version_nums[:3])
```

Comment:
Can we make this a full library function so others can use it and cache it value to the session?

Reply:
( or maybe there is a simpler way to not call this on every playback?)